validator/validator-main.protoascii

```diff
@@ -102,6 +102,12 @@ tags: {  # HTML tag for non-transformed AMP.
   tag_name: "HTML"
   mandatory: true
   mandatory_parent: "!DOCTYPE"
+  attrs: {
+    # This attribute should always be invalid. See https://github.com/ampproject/amphtml/pull/27174
+    name: "data-amp-autocomplete-opt-in"
+    value: "false"
+    blacklisted_value_regex: "false"
```

Comment:
This is a clever solution. Can you leave a comment here explaining what this is doing and why, because I suspect it'll be confusing to someone reading it in isolation.

Reply:
This `amp-autocomplete-opt-in` only applies to AMP4email even though it is defined on the `html` element shared across the various AMP flavors, correct?

Reply:
It is only relevant to the AMP4Email format, yes.